include/tig/blame.h

```diff
@@ -17,6 +17,8 @@
 #include "tig/view.h"
 
 extern struct view blame_view;
+extern char *filename_from_blame;
+extern unsigned long lineno_from_blame;
```

Comment:
I think creating these two globals isn't actually necessary. This information is also stored inside `blame_view`, though it's a bit fussy to extract it (see also a similar comment I made below).